lab-khalid/server.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const
+  express = require('express'),
+
+  morgan = require('morgan'),
+  cors = require('cors'),
+  debug = require('debug'),
```

Comment:
You need complete your debug statement here so that your debugs show up.
`debug = require('debug')('CFgram:server.js'),`